po/de.po

```diff
@@ -130,8 +130,9 @@ msgstr "Der Ausdruck %s ist falsch."
 
 #: src/string/i18n.c:15
 #, c-format
-msgid "The expression (as strings) %s is false."
-msgstr "Der Ausdruck (als Zeichenkette) %s ist falsch."
+msgid "The expression (as strings) %s is false: actual=`%s` expected=`%s`."
+msgstr ""
+"Der Ausdruck (als Zeichenkette) %s ist falsch: actual=`%s` expected=`%s`."
```

Comment:
@a1lu I'd propose "tatsächlich" instead of "aktuell" because aktuell is used 	in conjunction with time, while tatsächlich is used when talking about reality.

Example: [Was Verbraucher erwarten und was sie tatsächlich bekommen](http://www.heise.de/resale/artikel/Was-Verbraucher-erwarten-und-was-sie-tatsaechlich-bekommen-1845383.html)
roughly translated: What customers expect and what they actually receive